DigitalLearningSolutions.Web/Styles/home/welcome.scss

```diff
@@ -1,10 +1,32 @@
 @import "~nhsuk-frontend/packages/core/all";
 
+.welcome-page-button-container {
+  @include govuk-media-query($from: tablet) {
+    margin-bottom: 32px;
+  }
+
+  @include govuk-media-query($until: tablet) {

```

Comment:
Do we need both media queries? I think the standard is to write a default style first and then if necessary override that style with a media query (rather than have media queries for until tablet and after tablet). e.g.
```
.welcome-page-button-container {
  margin-bottom: 26px;

  @include govuk-media-query($from: tablet) {
    margin-bottom: 32px;
  }
}
```

Reply:
Yeah, I don't know the benefit of doing that, but happy to make that change.

Reply:
There was... some reason I did it this way but I can't remember what, so if it works fine without it that's probably better.